toolchain-m68k

```diff
@@ -241,6 +241,14 @@ def build():
                               path.join('{host}', 'bin'),
                               environ['PATH']])
 
+  """
+  When we have a working compiler in our path, we can also check if the
+  required headers/libraries are present.
+  """
+
+  require_header('<python2.7/python.h>')
+  require_header('<ncurses.h>', symbol = 'NCURSES_VERSION_MAJOR', value = 5)
```

Comment:
So either pass a desired error message to `require_header`, or let `require_header` return an error code and print and fail after the call if needed. Which is better?